app/jobs/email_csv_job.rb

```diff
@@ -0,0 +1,21 @@
+class EmailCsvJob < ApplicationJob
+  queue_as :default
+
+  BYTE_ORDER_MARK = "\uFEFF".freeze # Required to ensure Excel always reads CSV as UTF-8
+
+  EXPIRATION_TIME = 3.hours.to_i
+
+  def perform(user, search_term = nil, filters = {}, all_orgs = false, organisation = nil) # rubocop:disable Style/OptionalBooleanParameter
+    unfiltered_logs = organisation.present? && user.support? ? LettingsLog.all.where(owning_organisation_id: organisation.id) : user.lettings_logs
```

Comment:
Think the `.all` might be redundant here?

Reply:
Also not sure if the precedence might be a bit ambiguous, might be easier to parse with parenthesis?